pkg/integration/tests/patch_building/copy_patch_to_clipboard.go

```diff
@@ -0,0 +1,53 @@
+package patch_building
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+var BuildPatchAndCopyToClipboard = NewIntegrationTest(NewIntegrationTestArgs{
+	Description:  "Create a patch from the commits and copy the patch to clipbaord.",
+	ExtraCmdArgs: "",
+	Skip:         false,
+	SetupConfig:  func(config *config.AppConfig) {},
+	SetupRepo: func(shell *Shell) {
+		shell.NewBranch("branch-a")
+		shell.CreateFileAndAdd("file1", "first line\n")
+		shell.Commit("first commit")
+
+		shell.NewBranch("branch-b")
+		shell.UpdateFileAndAdd("file1", "first line\nsecond line\n")
+		shell.Commit("update")
+
+		shell.Checkout("branch-a")
+	},
+	Run: func(t *TestDriver, keys config.KeybindingConfig) {
+		t.Views().Branches().
+			Focus().
+			Lines(
+				Contains("branch-a").IsSelected(),
+				Contains("branch-b"),
+			).
+			Press(keys.Universal.NextItem).
+			PressEnter().
+			PressEnter()
+		t.Views().
+			CommitFiles().
+			Lines(
+				Contains("M file1").IsSelected(),
+			).
+			PressPrimaryAction()
+
+		t.Views().Information().Content(Contains("building patch"))
+
+		t.Views().
+			CommitFiles().
+			Press(keys.Universal.CreatePatchOptionsMenu)
+
+		t.ExpectPopup().Menu().Title(Equals("Patch Options")).Select(Contains("copy patch to clipboard")).Confirm()
+
+		t.ExpectToast(Contains("Patch copied to clipboard"))
```

Comment:
I've added an 'ExpectToast' function which we can use to know that we've copied the content to the clipboard as opposed to waiting

Reply:
Notice that I've changed the text that we're expecting. Could you update the text so that this test passes?

Reply:
sure . Made change